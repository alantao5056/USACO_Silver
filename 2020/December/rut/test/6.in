350
N 883724 156349
E 666793 817084
N 429643 429336
E 277392 833693
E 723190 837557
E 709943 824334
N 526110 263431
N 955126 331705
N 666316 313984
N 589339 357133
N 682684 240857
N 55179 907651
N 888176 853646
N 548634 131038
E 859594 854228
N 683402 80523
N 772480 606633
N 73077 78111
E 841726 744428
E 884665 850119
N 291530 532804
E 371097 104335
N 784101 508863
N 142792 573850
N 335177 433444
E 365118 116847
N 735746 889327
N 886157 962404
E 884625 320482
E 197077 205147
E 866662 13029
E 293456 900479
N 627902 684580
E 332345 827372
E 684179 678901
N 890241 44020
E 693178 296118
N 102749 182275
N 623136 583252
N 887950 780329
E 2884 646991
N 166974 456799
E 951751 84701
E 761744 933399
N 629234 617578
E 446521 24171
E 897007 717349
E 236820 336451
N 55389 959587
N 194440 847537
E 54990 850422
N 928570 533748
N 218980 1852
E 193971 763596
N 512686 909182
N 510302 872056
E 551939 769063
E 406045 5884
N 958779 61273
E 94764 772066
N 11079 343408
N 90736 271978
E 627193 7311
E 764704 201282
E 835613 713968
E 556805 224270
E 785815 292561
E 248527 698607
E 47690 173738
N 322792 268502
N 445432 795933
E 418493 886670
E 958007 30215
N 499373 794919
N 804335 146885
N 534832 703690
E 137384 5857
N 509000 770737
E 402852 818427
E 874330 141220
N 117873 586652
E 221177 521497
E 112812 995857
E 11757 495230
N 111985 815918
E 926180 867102
E 480998 4486
E 328378 513487
N 390716 432691
E 626793 307021
E 647724 941247
E 464588 678776
E 684475 791588
N 934577 803346
N 495950 915331
N 204209 357863
N 132509 838862
E 583019 167240
N 804266 74308
E 360293 701101
E 57539 865177
N 857790 329766
N 212242 14241
E 562551 948818
E 408090 444768
N 102252 165329
E 147163 814191
E 712913 397210
E 792150 201477
N 820431 561770
E 155733 135661
E 948634 509803
E 420311 722045
N 683577 284596
N 266541 209039
N 281188 311291
N 137347 458454
E 348719 171368
E 243684 479870
N 212957 300301
N 399683 972386
N 451587 437372
E 349609 374036
E 778224 573965
E 672009 840507
N 10563 638047
E 30798 775394
N 865224 124114
N 96860 367798
N 46326 97107
N 73913 496790
N 753514 464730
E 426567 330691
E 854369 108916
N 50415 297277
E 971853 307840
E 720933 338638
E 928468 720214
E 823723 817075
E 198849 379753
N 390628 453666
E 587497 723533
E 705387 150100
N 713665 520821
N 295909 571236
N 718659 59441
N 295809 296727
E 648354 225195
E 641364 48919
N 518525 764120
E 492792 671100
N 480202 258597
E 932342 480337
E 231526 194002
N 155977 489911
N 861354 724923
N 390924 537084
E 775484 701790
N 396610 343155
E 318460 378032
E 136414 387176
E 924387 383731
E 435893 316073
N 613851 63951
N 312630 219929
N 98998 81283
E 461159 988559
E 120047 280395
N 4130 677006
E 453485 511818
E 183311 648232
E 584238 88972
N 243762 524865
N 899975 655069
E 750190 967699
N 606669 583049
N 704423 44208
E 822910 164255
E 159539 168386
E 839323 138223
E 929843 837886
E 891187 422124
E 752615 182238
E 446468 82213
N 687920 348756
E 665654 471777
E 241023 176201
E 804951 515463
E 165627 675002
E 203309 30677
N 752809 960520
N 466894 368059
E 786302 120674
E 630576 567142
E 646271 771415
E 629666 953421
N 712462 194444
E 323114 999395
N 130726 681375
N 106724 884684
E 737297 153846
E 205757 137092
E 96341 923394
E 685515 70323
N 873255 716594
E 409751 862612
N 934377 575074
N 151555 414541
E 884197 61619
N 291840 168344
E 246513 905641
E 161705 111398
E 961309 724091
E 881588 409607
N 885361 282862
E 176901 208966
E 421862 659695
N 861267 811250
E 332771 211799
N 969299 19992
N 6863 266505
N 198273 944562
N 43127 905872
N 295023 303812
E 133595 189173
E 205113 366074
E 655595 787936
N 197504 165555
E 153954 498326
N 688376 983977
N 391141 507192
E 999548 221817
N 235706 264944
N 712307 76320
N 82163 726267
E 251233 931380
E 145622 103327
E 699705 300832
N 65039 971138
N 158525 659514
N 755650 567008
N 651985 566556
N 492882 318614
N 756003 30921
N 322936 629436
E 595394 397022
E 233926 542644
E 176746 242349
E 285121 823741
E 408174 982266
N 515205 254268
N 129429 422605
N 481822 915487
E 99159 187842
N 792817 27130
E 212741 622524
E 80398 372802
E 51382 549548
N 839550 351021
E 115208 759196
E 50381 790753
N 68914 920182
E 654004 918356
E 74060 17516
E 41389 810333
N 154507 23074
N 664647 103472
E 717431 671206
E 45269 27108
E 196847 142317
E 847044 709050
N 267861 777965
E 111283 431969
E 386545 22381
E 849650 63770
N 339885 218278
E 968996 399277
E 970910 633061
N 38725 678330
N 457561 391529
E 151945 754925
N 637082 539138
E 75106 650422
N 915135 36967
E 772679 402969
E 956909 742854
E 164213 228203
E 991341 199113
N 489676 754190
N 15120 728103
E 199232 396400
E 608747 33483
E 796227 108589
E 932028 540076
E 90210 312756
N 828515 786017
E 301449 950230
N 173661 941571
E 678934 431248
N 378889 962720
E 470374 161952
N 833136 770700
E 735036 566927
N 803711 498956
E 425705 105518
E 450668 934033
N 218486 235483
N 214507 925496
E 442346 604431
E 908023 983320
E 618155 970046
N 724688 319534
N 381056 570923
E 662366 374634
E 483752 316691
N 617894 767359
E 54405 985846
N 957498 716705
E 802162 159051
N 594427 583426
N 349411 717934
E 266348 442622
E 470337 340031
N 241254 2397
N 543830 486149
E 435573 620395
E 777702 191152
E 871665 148650
N 374773 467164
N 515016 61591
N 638580 927355
N 732190 193703
E 145334 180393
N 645754 937999
E 37159 481829
N 228904 433755
E 704360 211457
E 165662 599474
E 343669 490600
E 713432 5616
E 666622 644196
N 337745 376386
E 967843 38073
N 937972 200179
E 426850 237338
E 401877 466242
N 997299 686954
E 783873 852616
E 235449 712638
E 670542 426070
