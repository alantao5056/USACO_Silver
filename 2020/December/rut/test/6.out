18
0
1
0
0
0
0
0
28
0
0
0
20
22
18
0
0
0
0
1
15
0
0
0
36
0
89
2
34
10
0
0
78
0
0
0
10
0
0
0
3
0
1
0
0
0
0
2
0
6
0
0
1
34
0
56
0
0
0
0
0
0
0
0
0
5
19
27
2
0
41
46
0
44
9
1
0
0
0
0
4
4
0
0
2
0
0
41
0
1
0
0
0
0
0
0
0
2
0
56
0
2
1
82
0
3
0
0
17
6
0
0
0
2
1
14
0
0
0
2
0
3
0
5
0
0
1
16
1
0
0
1
0
6
0
0
1
0
8
1
0
77
7
0
26
3
0
0
0
0
0
0
0
0
0
1
0
0
1
1
0
1
1
1
0
0
67
0
0
5
0
0
0
14
1
0
1
0
0
0
1
0
0
0
0
0
0
43
0
4
0
0
1
1
3
84
0
0
0
0
0
0
5
2
0
43
0
0
1
14
7
0
20
0
32
0
0
14
0
3
0
0
0
31
0
7
0
9
2
68
0
0
0
0
2
0
8
0
0
0
2
2
2
1
33
0
1
0
37
0
2
7
4
5
0
16
0
11
39
1
3
0
0
0
0
1
3
11
0
0
17
6
0
0
0
0
0
0
0
0
1
0
6
5
0
3
0
0
0
0
30
0
1
0
1
2
0
6
0
1
18
4
3
4
0
0
2
0
12
0
86
0
0
33
27
1
4
2
10
74
36
34
0
5
6
0
0
0
48
17
83
9
0
85
4
3
1
0
0
0
1
2
4
0
0
2
19
0
0
0
